### DISCUSSION TITLE
Can't login(lazer version)

### DISCUSSION THREAD

**Discussion by weipoyen970211** [ppy/osu#22311] [Q&A]
Problem details:
I can't login in osu!(lazer)

Video or screenshot showing the problem:
[(https://1drv.ms/u/s!AvFm1eK8ZJ3_6X-mh2ZHzi6bZ605?e=hFbtcl)]

osu! version: osu! 2022.1228.0-lazer

**Comment by bdach** [comment]
Please state your platform and upload logs as per https://github.com/ppy/osu/discussions/15188.

Possible duplicate of https://github.com/ppy/osu/issues/17499.
